src/platforms/rcore_desktop_glfw.c

```diff
@@ -214,6 +214,9 @@ void ToggleFullscreen(void)
         WindowSizeCallback(platform.handle, CORE.Window.previousScreen.width, CORE.Window.previousScreen.height);
     }
 
+    // Refocus window
+    glfwFocusWindow(platform.handle);
```

Comment:
We should also test what happens when focus is lost while being in fullscreen/borderless fullscreen mode in general (like tabbing out to another window) and what happens when focus is gained again. 

Currently when I tab out of a fullscreen window (works fine) and then tab into it again it stays minimized on macOS.

There is callback for window focused in GLFW but I don't if it is used in raylib.

Reply:
FYI: [`WindowFocusCallback(GLFWwindow *window, int focused)`](https://github.com/raysan5/raylib/blob/32e9d882d7cfc5a8aa4e310a72131a92221b6553/src/platforms/rcore_desktop_glfw.c#L1746C1-L1750C2) is used in raylib but it only sets the focus flag